data/filters/youtube-clarification.yaml

```diff
@@ -0,0 +1,13 @@
+title: Hide Youtube channel clarification
+tags:
+  - youtube
+template: |
+  www.youtube.com###clarify-box > .ytd-watch-flexy.style-scope
+  www.youtube.com##div[id*="info-panel"]:has(div[id="content"]):has(div[class^="disclaimer-container"]):matches-path(/shorts)
```

Comment:
Upward actually works with arbitrary CSS selectors, both [in uBO](https://github.com/gorhill/uBlock/commit/72bb70056843024b1a31fe1ab9c90bd4e8260ba2) and [Adguard](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#extended-css-upward).

My understanding is that classes are cheap to match on, so the filter will exit faster if no disclaimer exist, instead of looping on all info-panel to check whether they have a disclaimer. Alternatively, `##ytd-shorts #info-panel:has(.disclaimer-container)` would be fine, for a shorter syntax to yours.

Reply:
`##ytd-shorts #info-panel:has(.disclaimer-container)` Will not work, because then the disclaimer-container has to be directly under info-panel, which is not the case. Or am I wrong again?